Chart.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
```

Comment:
What if we stick with the older API version and use new Bitnami charts in `requirements.yaml`? Wondering if that'll blow up.

Best case scenario this can give us both `2` and `3` Helm support for some time (as now).

Reply:
This line change is the reason why CI is not happy as it currently uses Helm 2.

Reply:
This part of the documentation could shed some light on this?
https://github.com/bitnami/charts/tree/master/bitnami/mongodb#to-1000

Reply:
@arms11 Understood, that helps.

@christopher-russell that means we should also switch CircleCI to run the Helm v3 for this PR:
https://github.com/StackStorm/stackstorm-ha/blob/master/.circleci/config.yml

Reply:
I have changed the orb to helm v3